pwchem/tests/tests_ligand_characterization.py

```diff
@@ -0,0 +1,185 @@
+# ***************************************************************************
+# *
+# * Authors:     Daniel Del Hoyo (daniel.delhoyo.gomez@alumnos.upm.es)
+# *
+# * This program is free software; you can redistribute it and/or modify
```

Comment:
This tests file is a copy of the tests_ligand_filtering.py, I guess you wanted to use it as template, which is fine, but you uploaded it directly and did not build the test for your protocol